secator/tasks/grype.py

```
@@ -60,7 +60,7 @@ class grype(VulnCode):
 		TIMEOUT: OPT_NOT_SUPPORTED,
 		USER_AGENT: OPT_NOT_SUPPORTED,
 	}
-	install_version = 'v0.91.2'
+	install_version = 'v0.115.0'
 	install_cmd_pre = {'*': ['curl']}
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)